config/settings/local.py

```diff
@@ -27,7 +27,7 @@
 
 INTERNAL_IPS = ['127.0.0.1', '10.0.2.2', ]
 
-if os.environ.get('USE_DOCKER') == 'yes':
+if os.path.isfile('/.dockerenv'):
```

Comment:
It can simplify this code a little bit but I'm worried that this feature is not documented in any way (officially) so it can easily change between docker versions. I would stick to the current solution.